lib/controller.js

```diff
@@ -604,17 +604,17 @@ controller.renameTessel = function(opts) {
 
 controller.printAvailableNetworks = function(opts) {
   opts.authorized = true;
-  return controller.standardTesselCommand(opts, function(tessel) {
-    log.info('Scanning for available networks...');
+  return controller.standardTesselCommand(opts, (tessel) => {
+    log.info('Scanning for visible networks...');
 
     // Ask Tessel what networks it finds in a scan
     return tessel.findAvailableNetworks()
-      .then(function(networks) {
-        log.info('Currently visible networks (' + networks.length + '):');
+      .then((networks) => {
+        log.info(`Found ${networks.length} network${networks.length > 1 ? 's' : ''} visible to ${tessel.name}:`);
```

Comment:
Clarifies that the list of wifi networks is determined by what the tessel can see, not what the host machine can see.